app/entry.server.tsx

```diff
@@ -7,7 +7,7 @@
 import { PassThrough } from "node:stream";
 
 import type { EntryContext } from "@remix-run/node";
-import { Response } from "@remix-run/node";
+import { createReadableStreamFromReadable } from "@remix-run/node";
```

Comment:
Remix dokumentasjon er litt vanskelig å forstå hva som skal til for å komme oss til v2

Fulgte denne https://x.com/BrooksLybrand/status/1704265835546578989?s=20